react-native-in-app-payments-quickstart/app/components/CardsOnFileCardView.tsx

```diff
@@ -29,21 +29,21 @@ CardsOnFileCardView.propTypes = {
   onSelectCardOnFile: PropTypes.func.isRequired,
 };
 
-function showConfirmation(cardOnFile, onConfirm) {
+function showConfirmation(cardOnFile: any, onConfirm: any) {
```

Comment:
can we use the explicit type when it's possible?

Reply:
Resolved it.